pkg/configs/client/configs_test.go

```diff
@@ -28,8 +29,11 @@ func TestJSONDecoding(t *testing.T) {
 		"2": {
 			ID: 1,
 			Config: configs.Config{
-				RulesFiles: map[string]string{
-					"recording.rules": ":scope_authfe_request_duration_seconds:99quantile = histogram_quantile(0.99, sum(rate(scope_request_duration_seconds_bucket{ws=\"false\",job=\"authfe\",route!~\"(admin|metrics).*\"}[5m])) by (le))\n",
+				RulesConfig: configs.RulesConfig{
+					Files: map[string]string{
+						"recording.rules": ":scope_authfe_request_duration_seconds:99quantile = histogram_quantile(0.99, sum(rate(scope_request_duration_seconds_bucket{ws=\"false\",job=\"authfe\",route!~\"(admin|metrics).*\"}[5m])) by (le))\n",
+					},
+					FormatVersion: configs.RuleFormatV2,
```

Comment:
The rules in recording.rules look to be in v1 format which is a bit confusing.

Reply:
Oh right, thanks! While this particular test doesn't care about this, I updated it for the humans :)

Reply:
Us humans appreciate it :)